recipes/pulseaudio/all/conanfile.py

```diff
@@ -1,7 +1,6 @@
 from conans import ConanFile, tools, AutoToolsBuildEnvironment, RunEnvironment
 from conans.errors import ConanInvalidConfiguration
 import os
-import glob
 
```

Comment:
```suggestion

required_conan_version = ">=1.29.0"
```

remove_files_by_mask: https://docs.conan.io/en/latest/changelog.html#id87